db/migrate/20190613095331_add_scenario_id_history_to_saved_scenarios.rb

```diff
@@ -0,0 +1,5 @@
+class AddScenarioIdHistoryToSavedScenarios < ActiveRecord::Migration[5.2]
+  def change
+    add_column :saved_scenarios, :scenario_id_history, :string
```

Comment:
This is super-nitpicky, but could we add `after: :scenario_id` here so that the column is created after the existing `scenario_id` column instead of `updated_at`?